metomi/isodatetime/data.py

```diff
@@ -612,58 +738,90 @@ def __str__(self):
             if prop_ == "days":
                 content_string += "T"
 
-        if content_string == "T":
-            # No content, zero duration.
-            content_string = "0Y"
-        elif content_string.endswith("T"):
+        if content_string.endswith("T"):
             # No time unit information, so strip the delimiter.
             content_string = content_string[:-1]
 
         total_string = start_string + content_string
         return total_string.replace(".", ",")
 
+    def __repr__(self):
+        return "<isodatetime.data.Duration: {0}>".format(repr(str(self)))
```

Comment:
Use `self` values instead of hard-coded values maybe? (see similar comment above)